.github/workflows/bleeding-edge.yml

```diff
@@ -46,10 +46,7 @@ jobs:
         run: sudo apt-get install libglu1-mesa-dev
       - name: Install build dependencies
         run: |
-          sudo apt-get install swig
-          python -m pip install -U pip setuptools wheel
-          python -m pip install numpy
```

Comment:
Removed `numpy` and `Cython` since those too are already mentioned in `pyproject.toml`, so it shouldn't be necessary to install them separately.